packages/manager/modules/pci/src/projects/project/data-processing/data-processing.html

```diff
@@ -0,0 +1,126 @@
+<div data-ui-view>
+    <oui-page-header data-heading="{{ ::'data_processing_jobs' | translate }}">
+        <oui-guide-menu
+            data-text="{{:: 'pci_project_guides_header' | translate }}"
+        >
+            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
```

Comment:
```suggestion
            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" external>
```
sometimes `data-` is omitted sometimes not. Could we keep consistency ? :)